src/BankService/Models/AccountState.cs

```diff
@@ -0,0 +1,10 @@
+namespace BankService.Models;
+
+public class AccountState
```

Comment:
AccountState should have transactions/ledger entries and not actual values for Balance.
Balance should be computed from the transactions